src/main/java/net/masterthought/cucumber/generators/EscapeHtmlReference.java

```diff
@@ -12,17 +12,17 @@
  */
 final class EscapeHtmlReference implements ReferenceInsertionEventHandler {
 
-    private static final PolicyFactory LINKS = new HtmlPolicyBuilder()
-            .allowStandardUrlProtocols().allowElements("a").allowAttributes("href")
-            .onElements("a").requireRelNofollowOnLinks().requireRelsOnLinks("noopener", "noreferrer")
+    private static final PolicyFactory TAGS = new HtmlPolicyBuilder()
+            .allowStandardUrlProtocols().allowElements("a", "span", "img").allowAttributes("href", "onclick", "src", "id", "style")
+            .onElements("a", "span", "img").requireRelNofollowOnLinks().requireRelsOnLinks("noopener", "noreferrer")
             .toFactory();
```

Comment:
`.allowAttributes("href", "onclick", "src", "id", "style").onElements("a", "span", "img")` 

This doesn't look right at all. Allowed attributes should be set for each individual element not for the whole group.